lib/skate_web/channels/vehicles_channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule SkateWeb.VehiclesChannel do
+  use SkateWeb, :channel
+
+  def join("vehicles:" <> route_id, _message, socket) do
+    socket = assign(socket, :route_id, route_id)
```

Comment:
Do you need the route_id? I don't see it used elsewhere.

Reply:
Nope, doesn't look like it.